dreem/datasets/sleap_dataset.py

```
@@ -21,8 +23,9 @@ def __init__(
         self,
         slp_files: list[str],
         video_files: list[str],
+        data_dirs: list[str] = [],
```

Ruff (0.7.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)